docker_signal.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Is there an actual reason for this file? Don't you normally shut down docker via `docker stock <container name>`?